scripts/mission_commander.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+import rospy
+import argparse
+import roslaunch
+import tf
+from std_msgs.msg import Int32
+from robosar_messages.srv import *
+from robosar_messages.msg import *
+from mtsp_ta import MtspCommander
+from greedy_ta import GreedyCommander
+
+
+class MissionCommander:
+
+    def __init__(self, args):
+        rospy.init_node('mission_commander', log_level=rospy.DEBUG)
+        rospy.logdebug("Initializing Mission Commander ...")
+        rospy.Subscriber('/mission_status', Int32, self.handle_commands)
+        self.args = args
+        rospy.set_param('/make_graph', self.args.make_graph)
+        rospy.set_param('/graph_name', self.args.graph_name)
+        rospy.set_param('/home_positions', [[45, 10], [49, 11], [50, 10]])
+        self.launch_mission = False
+        self.stop_mission = False
+        self.home_mission = False
+        self._tc_node = None
+        self._launch = roslaunch.scriptapi.ROSLaunch()
+        self._tc_process = None
+
+    def mission_main(self):
+        rate = rospy.Rate(10)
+        while not rospy.is_shutdown():
+            if self.launch_mission:
+                self.execute()
+                self.launch_mission = False
+            elif self.stop_mission:
+                self.stop()
+                self.stop_mission = False
+            elif self.home_mission:
+                self.homing()
+                self.home_mission = False
+            rate.sleep()
+
+    def handle_commands(self, msg):
+        if msg.data == 1:
+            self.launch_mission = True
+        elif msg.data == 2:
+            self.stop_mission = True
```

Comment:
Would have also been nice if you would have defined a custom message (with an enum) for this topic instead of using an Int, so that you dont have to look into the code to see what each integer means while using it from the command line